pylint/test/acceptance/test_stdlib.py

```diff
@@ -0,0 +1,25 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+import os
+
+import pytest
+
+import pylint.lint
+
+LIB_DIR = os.path.dirname(os.__file__)
+MODULES_TO_CHECK = [f for f in os.listdir(LIB_DIR) if f.endswith(".py")
+                    or os.path.exists(os.path.join(LIB_DIR, f, '__init__.py'))]
+MODULES_TO_CHECK = MODULES_TO_CHECK
```

Comment:
@degustaf I had a slice there to limit module count during test runs when implementing it. Obviously it needs to be deleted.